apps/molgenis-components/src/client/client.ts

```diff
@@ -38,20 +38,23 @@ const client: IClient = {
       },
       fetchTableData: async (
         tableId: string,
-        properties: IQueryMetaData = {}
+        properties: IQueryMetaData = {},
+        expandLevel: number = 1
       ) => {
         const schemaMetaData = await fetchSchemaMetaData(schemaId);
-        return fetchTableData(tableId, properties, schemaMetaData);
+        return fetchTableData(tableId, properties, schemaMetaData, expandLevel);
       },
       fetchTableDataValues: async (
         tableId: string,
-        properties: IQueryMetaData = {}
+        properties: IQueryMetaData = {},
+        expandLevel: number = 1
       ) => {
         const schemaMetaData = await fetchSchemaMetaData(schemaId);
         const dataResp = await fetchTableData(
           tableId,
           properties,
-          schemaMetaData
+          schemaMetaData,
+          expandLevel
```

Comment:
why not add expandLevel as a typed prop , instead of using positional props, this applies for of the usage in this pr

Reply:
Makes sense. Can you or @chinook25 make the necessary changes?

Reply:
done